.github/dependabot.yml

```diff
@@ -7,8 +7,6 @@ updates:
       day: "thursday"
     labels:
       - "dependencies"
```

Comment:
Can you remove your changes to dependabot? I've got a separate PR I'm going to merge in with changes.